### PR TITLE
Apache Flink tutorial as playbook

### PR DESCRIPTION
**Pull request by ftisiot**
# What changed, and why it matters

I fixed some steps of the Flink playbook for the set of parameters that are shown now and to have a more consistent set of steps.
I also hidden the SQL needed to generate the table/job under a dropdown and changed the wording to have people trying to attempt before checking the pre-built solution.

I'm looking for feedback if this is the right thing, rather than just a "WRITE THIS STATEMENT HERE" approach

As of now the changes are viewable at https://deploy-preview-551--developer-aiven-io-preview.netlify.app/docs/products/flink/howto/real-time-alerting-solution.html

If ok I'll also drag them to the CLI version https://deploy-preview-551--developer-aiven-io-preview.netlify.app/docs/products/flink/howto/real-time-alerting-solution-cli.html

### CODE REVIEW

**Comment by TibsAtWork** [comment]
✅ I do like the "gamifying" of the article by allowing the user to try to work out the SQL themselves (if they want) and then see the answer. It also makes it easier to skim the article if one wants to just get a general idea of what it is about.

The rest of this comment is just fiddling with details.

I'm not 100% sure I'd have immediately figured out what to do to see the SQL in the dropdown.

I'm not sure I know how best to fix that (for me). I *think* I'd think about changing the lead text to be more like:
```
What should the XXX be? Try it yourself or click below to reveal the solution.
```

(note I've also moved the `be` to the end of its sentence, and added an `it` after `Try`).

**Comment by ftisiot** [comment]
Thanks @lornajane! 
I'll change the errors mentioned.

I agree, the CLI was a leftover from the initial docs. The [blog post we're about to release](https://github.com/aiven/blog-posts/pull/38) shows all the needed CLI steps so I think we can remove it.

This open a new question: can we just remove an existing doc? it will break links, if they're in devportal we're ok, but the others might be a problem

**Comment by lornajane** [comment]
Can we open a separate issue to remove the CLI version and work around the links? We'll need this some time and sphinx does have a redirects thing, I just haven't used it yet.

**Comment by ftisiot** [comment]
Created #565

**Comment by ftisiot** [comment]
@TibsAtWork moved away from dropdowns, this is ready for review now

File: docs/products/flink/howto/timestamps_opensearch.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+When the result of the data pipeline contains a timestamp column like the below:
+
+::
+
+    EVENT_TIME TIMESTAMP(3),
+    HOSTNAME STRING,
+    CPU DOUBLE
+
+To push the data correctly to an OpenSearch® index, you'll need to set the target column format as ``STRING`` in the Flink table definition, like:
```
Comment (TibsAtWork) [inline docs/products/flink/howto/timestamps_opensearch.rst:27]: Should this be lowercase `t` at the start of the line, as the text is continuing? I'm really not sure...

(I read this text to myself as  if it were `When <condition>, to push the data correctly ...`)

File: docs/products/flink/howto/timestamps_opensearch.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+::
+
+    EVENT_TIME STRING,
+    HOSTNAME STRING,
+    CPU DOUBLE    
+
+And, assuming the ``EVENT_TIME`` is a timestamp, you'll need to specify it in the format understood by OpenSearch® using the ``DATE_FORMAT`` function, like:
```
Comment (TibsAtWork) [inline docs/products/flink/howto/timestamps_opensearch.rst:35]: Similarly, should this be lowercase `a` at the start of the line, as the text is continuing?
For this one, I think I would actually remove the `And,` and start the sentence with `Assuming`.

**Review by TibsAtWork — COMMENTED** [review]
I've done some englishing to the text. and made a couple of comments.

I did do some minor changes to the CLI text to bring it more in-line with the UI text - mainly removing the OpenSearch references that aren't followed up by an example, and a little bit of consistency in the section titles.

I haven't yet followed the UI text, to see if it works - I don't know if someone else has already done that (I shall do so anyway, for my own satisfaction and learning).

Provided the directions are accurate, and once you've decided on my couple of outstanding comments, I'd say this is pretty much ready.

**Comment by ftisiot** [comment]
Hi @TibsAtWork , thanks for this. 
The setup has been tested before (only the Opensearch bit is missing)

**Comment by TibsAtWork** [comment]
@ftisiot I've now worked through all of the instructions.
I've:

- fixed a bug in the SQL for the fourth example,
- changed some `select` to be `enter`, 
- changed the names of some of the fields to be edited in the Data Tables / Kafka to match more nearly the actual field names - for instance **kafka** instead of the longer text that was there before.
- and made a few other minor changes.

If the changes look OK to you, I'm happy to approve.

**Comment by TibsAtWork** [comment]
(OK - made vale happy by capitalising the field names, but the main change needed was for kafka anyway)

**Comment by ftisiot** [comment]
LGTM fixed a small issue, should be good to go

**Comment by TibsAtWork** [comment]
Thanks for spotting that!

**Review by TibsAtWork — APPROVED** [review]
Now believed ready to go

**Pull request merged**